src/META-INF/plugin.xml

```diff
@@ -26,7 +26,7 @@
     <p>Alpha pre-release of the 1.0.0 version.</p>
     <p>Doesn't contain all the functionality of the 0.9.x branch but has a completely reworked internals.
       It's faster than 0.9.x, refactoring works to some degree and has native support for gopath packages.</p>
-    
+
```

Comment:
Sorry, that's my default setting to remove all trailing spaces from all lines when saving files which seems it's not in this project settings. I'll see if I can change the commit somehow.
Maybe it's a good idea to create PR to cleanup the sources and remove unused imports as well?